test/auro-datepicker.test.js

```diff
@@ -31,17 +31,6 @@ describe('auro-datepicker', () => {
     await expect(el.shadowRoot.activeElement).to.be.equal(input);
   });
 
```

Comment:
test `'can preset a value'` fails

Reply:
Issue was on my side, I had a global link to input and menu before we made the update.